modules/internal/LocaleModelHelpNUMA.chpl

```diff
@@ -151,16 +155,22 @@ module LocaleModelHelpNUMA {
     //
     const dnode =  chpl_nodeFromLocaleID(loc);
     const dsubloc =  chpl_sublocFromLocaleID(loc);
+    var tls = chpl_task_getChapelData();
+    var isSerial = chpl_task_data_getSerial(tls);
     if dnode == chpl_nodeID {
-      if __primitive("task_get_serial") then
```

Comment:
It's still used, but anybody using it could just call `chpl_task_getSerial` directly since it's a Chapel function. I'd guess there's some benefit to leaving it a primitive so the compiler can analyze it, but I don't know if we actually do that. Anyway I intend to leave it alone for this PR at least.